docs/source/guide/s3-example-creating-buckets.rst

```diff
@@ -61,6 +77,8 @@ or to address regulatory requirements.
         return True
 
 
+
+
```

Comment:
Were these lines intentional? We usually wouldn't have 4 new lines after an example.

Reply:
Thanks for pointing it out. No they weren't intentional. Will fix it.